### PR TITLE
CLDR-14332 correct link

### PR DESCRIPTION
**Pull request by FrankYFTang**
CLDR-14332

- [ ] This PR completes the ticket.

### CODE REVIEW

**Review by macchiati — APPROVED** [review]

**Comment by AEApple** [comment]
@srl295 - Associated this fix with the ticket currently assigned to you since it seems like the closest.

File: docs/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ CLDR main page: [https://unicode.org/cldr](unicode.org/cldr)
 
 ## TR35
 
-- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](ldml/tr35.html)
+- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](https://unicode-org.github.io/cldr/ldml/tr35.html)
```
Comment (srl295) [inline docs/README.md:7]:
```suggestion
- [Unicode Technical Standard #35 UNICODE LOCALE DATA MARKUP LANGUAGE (LDML)](ldml/tr35.md)
```

I don't think we should make this change. This should be a relative link. It's a html link as a compromise until #1572 lands.

**Review by srl295 — CHANGES_REQUESTED** [review]
What is the intention?

**Comment by macchiati** [comment]
Ok, let's close

**Comment by srl295** [comment]
no reply, closing

**Pull request closed without merging**